grapple/actions.py

```diff
@@ -468,7 +468,8 @@ class Meta:
 
             # Add support for `graphql_fields`
             methods["resolve_" + field.field_name] = (
-                custom_cls_resolver(cls=cls, graphql_field=item) or streamfield_resolver
```

Comment:
I'd really like to see a regression test covering this specific change and to add comments, reflecting the annotations in the OP, to cls.graphql_fields clarifying that item can be either a GraphQLField or tuple, and to update the comment above get_field_type to indicate that it is also resolving item into a GraphQLField.

Reply:
I tested removing the item change and it broke. I'll call that good enough.